build/build/src/enso.rs

```diff
@@ -74,7 +74,7 @@ impl BuiltEnso {
 
     pub async fn run_benchmarks(&self, opt: BenchmarkOptions) -> Result {
         self.cmd()?
-            .with_args(["--run", self.paths.repo_root.test.benchmarks.as_str()])
+            .with_args(["--jvm", "--run", self.paths.repo_root.test.benchmarks.as_str()])
```

Comment:
The benchmarks are run in JVM mode, but this is not the default? Is there a risk that the measurements won't accurately reflect typical user experience?

Reply:
Yup. The _native image_ binary is only generated when one invokes `engine-runner/buildNativeImage`. That's **not** happening **when building production** bits. However we do have CI job that verifies the `engine-runner/buildNativeImage` works. The job also runs benchmarks - but benchmarks need to run in `--jvm` mode. Hence this change.

Reply:
Moreover this is not really running the benchmarks, it just verifies the benchmarks are "executable". The actual benchmarks run is happening [elsewhere](https://github.com/enso-org/enso/actions/workflows/benchmark.yml).